app/src/js/applications/codelists/routes/index.js

```diff
@@ -1,19 +1,27 @@
-import React  from 'react';
+import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 
-import { Home } from 'bauhaus-codelists';
-
-import D from 'js/i18n';
+import {
+	Menu,
+	Home,
+	CodelistComponentView,
+	dictionary,
+} from 'bauhaus-codelists';
 
 const CodesListComponent = () => {
-	console.log("yo")
-	document.title = 'Bauhaus - ' + D.structuresTitle;
+	document.title = 'Bauhaus - ' + dictionary.codelistsTitle.fr;
```

Comment:
You must not use the french version. You muse use the "D" object like here : https://github.com/InseeFr/Bauhaus/blob/master/app/src/js/applications/operations/routes/index.js#L20

It will automatically select the right dictionary based on the language of the browser.